gulpfile.js

```diff
@@ -0,0 +1,61 @@
+var gulp = require('gulp');
+var concat = require("gulp-concat");
+var uglify = require("gulp-uglify");
+var cleanCSS = require('gulp-clean-css');
+
+gulp.task('minify-all', ['vendor-styles', 'vendor-fonts', 'vendor-images', 'scripts']);
+
+gulp.task('vendor-styles', function () {
+    gulp.src([
+        './source/vendor/font-awesome/css/font-awesome.min.css',
+        './source/vendor/open-sans/styles.css',
+        './source/vendor/source-code-pro/styles.css',
+        './source/vendor/lightgallery/css/lightgallery.min.css',
+        './source/vendor/justified-gallery/justifiedGallery.min.css'
+        ])
+        .pipe(concat('vendor-styles.min.css'))
+        .pipe(cleanCSS())
```

Comment:
could you use these options `.cleanCSS({compatibility: 'ie9', processImport: false, keepSpecialComments: 0})`